packages/cms-lib/lib/config.js

```diff
@@ -139,10 +142,10 @@ const isConfigPathInGitRepo = () => {
 
 const configFilenameIsIgnoredByGitignore = ignoreFiles => {
   return ignoreFiles.some(gitignore => {
-    const gitignoreContents = fs.readFileSync(gitignore).toString();
+    const gitignoreContents = fs.readFileSync(gitignore);
```

Comment:
Should this be changed? I thought that `gitignoreContents` needed to be a `string` not a `Buffer`.

Reply:
I changed this back. Strange how this works on my mac, but not on windows.